src/pages/workspaces/workspace/analysis/cost-utils.ts

```diff
@@ -166,20 +192,23 @@ export const getAzureDiskCostEstimate = ({ region, diskSize }) => {
 
 export const getPersistentDiskCostMonthly = ({ cloudContext = {}, diskType, size, status, zone }, computeRegion) => {
   const price = Utils.cond(
+    // @ts-expect-error
```

Comment:
I put a few expect errors here because this code is used in a decent amount of untested locations, and the types added already required a lot of places to be updated. This code is a high-risk area that we plan to add types to soon (and it is already decently well tested isolation)